examples/customize/build_graph/pipeline/text_to_lexical_graph_to_entity_graph_single_pipeline.py

```diff
@@ -0,0 +1,196 @@
+"""In this example, we implement a single pipelines with two neo4j writer to
+deal with the lexical graph (Document and Chunks) and the entity graph (entities and
+relations extracted from the text)
+"""
```

Comment:
Maybe change this to something like:
"In this example, we set up a single pipeline with two Neo4j writers: one for creating the lexical graph (Document and Chunks) and another for creating the entity graph (entities and relations derived from the text)."

Also should this be a comment rather than a string?

Reply:
Thank you, description updated. Regarding comment VS docstring, at the moment we have docstrings in all examples, so I'll stick to that "convention" for now, but we can update all of them later.